repo/content/content_manager_test.go

```diff
@@ -1809,22 +1809,24 @@ func (s *contentManagerSuite) TestAutoCompressionOfMetadata(t *testing.T) {
 	data := blobtesting.DataMap{}
 	st := blobtesting.NewMapStorage(data, nil, nil)
 	bm := s.newTestContentManager(t, st)
+	verifyMetadataCompression(ctx, t, bm, compression.HeaderZstdFastest)
```

Comment:
IIUC, this is the test case where `mp.MetadataCompression == ""`, right? The explicit algo case and the `"none"` case are tested below

Reply:
That's right